templates/rst_display.html

```diff
@@ -5,6 +5,9 @@
 
 {% block title %}{{ rst.title }}{% endblock %}
 
+{% block og_url %}{{ [config.canonical_url, ctx.slug, '/']|join('') }}{% endblock %}
```

Comment:
I played with `link_to('page', slug=ctx.slug)` with some success, but the above approach seemed to work just as well if not better. In any case, I don't think either is really what we want.